server/src/main/java/org/opensearch/search/profile/query/ConcurrentQueryProfileBreakdown.java

```diff
@@ -202,6 +202,11 @@ Map<Collector, Map<String, Long>> buildSliceLevelBreakdown() {
                         // doesn't represent an actual timing; rather, it indicates no invocations.
                         continue;
                     }
+                    if (sliceLeafTimingTypeStartTime == 0L && currentSliceBreakdown.get(timingTypeCountKey) == 0L) {
```

Comment:
This check and the one above on line 197 is essentially ignoring the leaf with 0 invocation for a timing type. We can merge these and instead add a check if the timingTypeCount is 0 then ignore that leaf timingType values.

Reply:
Hi @sohami, thanks for the review. It's been addressed now.